task_queue/db.py

```diff
@@ -0,0 +1,73 @@
+import chromadb
+from chromadb.config import Settings
+
+def get_connection():
```

Comment:
Can we create a class instead, with all the initialisations in the __init__ function. This will eliminate need to call get_connection for creating the function. We can then create the CRUD operations inside the class.

@UB-AICLUB Thoughts?

Reply:
yeah we could go with a class but i guess lets not make him do this over again this works na so lets go ahead with it